src/applications/mhv-medical-records/components/LabsAndTests/EkgDetails.jsx

```diff
@@ -87,13 +87,15 @@ const EkgDetails = props => {
         className="vads-u-margin-bottom--0"
         aria-describedby="ekg-date"
         data-testid="ekg-record-name"
+        data-dd-privacy="mask"
       >
         {record.name}
       </h1>
       <DateSubheading
         date={record.date}
         id="ekg-date"
         label="Date and time collected"
+        data-dd-privacy="mask"
       />
```

Comment:
`DateSubheading` already has a mask inside that component so you don't need to add it here.

Reply:
fixed.